be/src/exec/pipeline/scan_operator.cpp

```diff
@@ -129,12 +129,14 @@ void ScanOperator::_trigger_next_scan(RuntimeState* state) {
     };
     // TODO(by satanson): set a proper priority
     task.priority = 20;
-    if (!_io_threads->offer(task)) {
-        LOG(WARNING) << "ScanOperator failed to offer io task due to thread pool shutdown";
+    if (!_io_threads->try_offer(task)) {
+        LOG(WARNING) << "ScanOperator failed to offer io task due to thread pool overload";
```

Comment:
when thread pool is overloaded? and what will happen if that? 

should not we wait ?

Reply:
We better not wait here, because it will reduce pipeline's efficiency(the number of pipeline's thread is fixed, usually equals with core size). But we can retry it next time

Reply:
So you have handled `Status::RuntimeError` right? Is there any another status like 'RetryLater' or something?